pkg/apis/v1alpha1/provider.go

```diff
@@ -117,6 +117,15 @@ type MetadataOptions struct {
 	// 1.0 credentials are not available.
 	// +optional
 	HTTPTokens *string `json:"httpTokens,omitempty"`
+
+	// InstanceMetadataTags allow access to instance tags from the instance metadata.
+	// By accessing tags from the instance metadata, you no longer need to use the DescribeInstances
+	// or DescribeTags API calls to retrieve tag information, which reduces your API transactions per second,
+	// and lets your tag retrievals scale with the number of instances that you control. Furthermore,
+	// local processes that are running on an instance can view the instance's tag information directly
+	// from the instance metadata. This is disabled by default.
+	// +optional
+	InstanceMetadataTags *string `json:"instanceMetadataTags,omitempty"`
```

Comment:
We've tried supporting this at one point but found that this failed due to the lack of support for `/` characters from the IMDS service. We've tried to communicate with the team that this this is a feature that users want, but this still seems to be unsupported.

Reply:
Sorry, I wasn't aware of that limitation, but thank you for the backstory! I will add my use case on that feature as well so hopefully it receives some precedence!

Reply:
Ok, great! I'm going to close this PR for now since I don't expect that we will be able to make progress on this PR without support from the EC2 IMDS side.

Reply:
Sounds good, is there any way that I can support raising this to the AWS team? If you need any support triaging issues, feel free to send them my way